packages/core/content-manager/server/controllers/collection-types.js

```diff
@@ -183,6 +183,17 @@ module.exports = {
 
     const clonedEntity = await entityManager.clone(entity, sanitizedBody, model);
 
+    const mediaFields = Object.keys(model.attributes).filter((attributeName) => {
+      const attribute = model.attributes[attributeName];
+      return attribute.type === 'media';
+    });
+
+    for (const mediaField of mediaFields) {
+      if (entity[mediaField]) {
+        clonedEntity[mediaField] = entity[mediaField];
+      }
+    }
```

Comment:
Good catch, indeed at the moment polymorphic relations (the ones used for media) are not being clonded.

shouldn't the media be added before cloning the entity? What I am seeing here is the media attribute is added into the response, but it's not actually saved in db.

Reply:
Wait ignore the latest commit, I am still fixing the code.

Reply:
great! I will be testing the code tomorrow morning, expect this to be merged before next week release